frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -187,6 +189,19 @@ function PlotlyFigure({
     return spec
   }, [element, figure.spec, theme, widgetMgr])
 
+  // This is required for the form clearing functionality:
+  React.useEffect(() => {
+    const formClearHelper = new FormClearHelper()
+    formClearHelper.manageFormClearListener(
+      widgetMgr,
+      element.formId,
+      getInitialValue
```

Comment:
hmm, not sure if that would work as expected. `getInitialValue` doesn't really reset the selections in the chart or?

Reply:
I believe you could use the `reset` callback method that is defined below

Reply:
Yeah I thought reset would do it which it works for any box selects which I tried out yesterday. However, it doesn't work with single selections soi turned it into draft. Will need to investigate more.